UVAtlas/isochart/UVAtlas.cpp

```diff
@@ -270,6 +270,7 @@ namespace
 
         if (!adjacency)
         {
+            wprintf(L"Input adjacency pointer cannot be nullptr. Use DirectXMesh to compute it\n");
```

Comment:
Can you remove all the debug edits here?

Note that SysInternals DBGVIEW.EXE is a great way to see the debug output without a full IDE.